### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by HunBurry** [scotch-io/node-web-scraper#5]
I was trying to read the tutorial, but I did not understand it fully. I am new to js and node, and was wondering if I could get some help on the code itself. I found the url I would like to use, and subbed out the variable names. I have also went in and found the info I need in the page, and where it was in the scripting. However, this is as far as I could get. Any help after this would be appreciated.

**Comment by crashspringfield** [comment]
Were you able to get the example to work following it exactly?

**Comment by jpca999** [comment]
nope, i tried running the server,  node server.js  and theres nothing coming down the wire, nothing in console  Also in the browser it says'  

cannot get localhost:80801
![image](https://cloud.githubusercontent.com/assets/1026325/23146437/bd9c6a7a-f7a4-11e6-94c6-c37abf3a19c6.png)

**Comment by jpca999** [comment]
lol - got it working  at http://localhost:8081/scrape

**Comment by alee092017** [comment]
Thanks @jpca999 this was driving me nuts.  all my code was correct, selectors, everything.  facepalm.
